app/scripts/copy-project.ts

```diff
@@ -123,7 +123,14 @@ await Promise.all([
     "DataChannel",
     `"destinationId" IN (SELECT id FROM "Node" WHERE "projectId" = '${projectId}')`,
   ),
-  copyTable("NodeEntry", `"nodeId" IN (SELECT id FROM "Node" WHERE "projectId" = '${projectId}')`),
```

Comment:
Supercedes https://github.com/OpenPipe/OpenPipe/pull/661?

Reply:
Forgot to merge that in, thanks